src/sentry/api/endpoints/organization_fork.py

```diff
@@ -0,0 +1,158 @@
+import logging
+from string import Template
+
+from django.db import router
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import Endpoint, region_silo_endpoint
+from sentry.api.endpoints.relocations.index import (
+    get_autopause_value,
+    validate_relocation_uniqueness,
+)
+from sentry.api.permissions import SuperuserOrStaffFeatureFlaggedPermission
+from sentry.api.serializers import serialize
+from sentry.hybridcloud.services.organization_mapping import organization_mapping_service
+from sentry.models.relocation import Relocation
+from sentry.organizations.services.organization import organization_service
+from sentry.tasks.relocation import uploading_start
+from sentry.types.region import get_local_region
+from sentry.utils.db import atomic_transaction
+
+ERR_DUPLICATE_ORGANIZATION_FORK = Template(
```

Comment:
Will this api endpoint always be gated by superuser/staff? Curious because the wording around `fork` is pretty specific to developers and non-technical people may be confused by this terminology

Reply:
Yes, this process is not intended to be self-serve for the foreseeable future. If we ever change that decision, we can revisit the naming and/or make an alias API endpoint.